bugsnag-android-performance/src/main/kotlin/com/bugsnag/android/performance/SpanEndCallback.kt

```diff
@@ -0,0 +1,5 @@
+package com.bugsnag.android.performance
+
+public fun interface SpanEndCallback {
```

Comment:
/**
 * Called whenever a [Span] is ending, but before it has been enqueued for delivery. These callbacks
 * can modify the attributes of a span or cause it to be discarded between the time [Span.end] is called
 * and when the span is enqueued for delivery.
 */